apps/web/e2e/unit-tests-converted/__tests__/integration/audio-delay-compensation.test.ts

```
@@ -0,0 +1,853 @@
+              
+              return {
+                success: true,
+                previousState,
+                undoneState: currentState,
+                historyLength: history.length,
+              };
+            },
+
+            canUndo: () => history.length > 1,
+            getHistory: () => [...history],
+            clearHistory: () => history.length = 0,
```

Biome (2.1.2)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)